README.md

```diff
@@ -37,3 +38,7 @@ SpiNNaker.  Tools exist for:
  - Clock discipline
  - Routing table minimisation
  - Optimal routing key allocation
+
+# Documentation
```

Comment:
Will attempt to do this and also incorporate the changes from the development-docs branch in a PR coming soon to this repository.